packages/pancake-uikit/src/components/DropdownMenu/DropdownMenu.tsx

```diff
@@ -20,110 +19,42 @@ const DropdownMenu: React.FC<DropdownMenuProps> = ({
   showItemsOnMobile = false,
   activeItem = "",
   items = [],
-  openMenuTimeout = 0,
   index,
   setMenuOpenByIndex,
   ...props
 }) => {
   const [isOpen, setIsOpen] = useState(false);
   const [targetRef, setTargetRef] = useState<HTMLDivElement | null>(null);
   const [tooltipRef, setTooltipRef] = useState<HTMLDivElement | null>(null);
-  const hideTimeout = useRef<number>();
-  const isHoveringOverTooltip = useRef(false);
   const hasItems = items.length > 0;
-  const clickTimeRef = useRef(openMenuTimeout);
   const { styles, attributes } = usePopper(targetRef, tooltipRef, {
+    strategy: "fixed",
     placement: isBottomNav ? "top" : "bottom-start",
     modifiers: [{ name: "offset", options: { offset: [0, isBottomNav ? 6 : 0] } }],
   });
 
   const isMenuShow = isOpen && ((isBottomNav && showItemsOnMobile) || !isBottomNav);
 
-  /**
-   * See "useTooltip"
-   */
   useEffect(() => {
-    const showTooltip = (evt: MouseEvent | TouchEvent) => {
+    const showTooltip = () => {
```

Comment:
Can you please rename showTooltip & hideTooltip to showDropdownMenu & hideDropdownMenu? I think current names is some copy paste leftovers from tooltip component 😬

Reply:
done 2a0aff8